### ISSUE TITLE
false positive CVE-2002-1647 for slash npm package

### ISSUE THREAD

**Issue by thediveo** [anchore/grype#446]
**What happened**:

Running grype on a syft-generated BOM for github.com/thediveo/lxkns raises CVE-2002-1647. However, CVE-2002-1647 relates to a web site but not an npm package.

**What you expected to happen**:

No false positive of CVE-2002-1647 for npm module.

**How to reproduce it (as minimally and precisely as possible)**:

Checkout github.com/thediveo/lxkns. Then run grype on newly checked-out repository directory.

**Environment**:
- grype build from 2dd41311cb250a621db615f5f7085edda88e4bb1
- go1.17.1
- linux/arm64

**Comment by luhring** [comment]
Good catch! I see no sign of https://nvd.nist.gov/vuln/detail/CVE-2002-1647 being related to https://www.npmjs.com/package/slash.

@wagoodman It looks like another case of false positive via `*` vendor:

```json
"matchDetails": [
  {
    "matcher": "javascript-matcher",
    "searchedBy": {
      "namespace": "nvd",
      "cpes": [
        "cpe:2.3:a:*:slash:3.0.0:*:*:*:*:*:*:*"
      ]
    },
    "found": {
      "versionConstraint": "none (unknown)",
      "cpes": [
        "cpe:2.3:a:slashcode.com:slash:*:*:*:*:*:*:*:*"
      ]
    }
  }
],
```

**Comment by patrick330602** [comment]
This is still being reported as a vulnerability in slash 3.0.0. Is there any update on this?

**Comment by OfriOuzan** [comment]
We encountered the same issue on the following environment
**What happened**:
In a Vulnerability Scanner Benchmark Research we are conducting, we executed Grype on 20 different containers and found out that Grype has multiple False Positives.
**What you expected to happen**:
We expected Grype not to report on these CVEs.
**How to reproduce it (as minimally and precisely as possible)**:
Install the Docker Images (from the links below) and execute Grype using the following command:
`grype <container_name> —-output json > <output_file_path>`

- **Output of `grype version`**:
Application:          grype
Version:              0.41.0
Syft Version:         v0.50.0
BuildDate:            2022-07-06T15:20:18Z
GitCommit:            0e0a9d9e7a28592db489499db0294608e5fe69b8
GitDescription:       v0.41.0
Platform:             linux/amd64
GoVersion:            go1.18.3
Compiler:             gc
Supported DB Schema:  4

**Kibana**
 
- **Container Details**:
https://hub.docker.com/layers/kibana/library/kibana/8.3.2/images/sha256-51635619b14a0f3a764f39c4c51d527304d8c33fbda05d72652b18255639122b?context=explore
 
- **OS (e.g: `cat /etc/os-release`)**:
NAME="Ubuntu"
VERSION="20.04.4 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.4 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
- **CVEs**
**CVE-2002-1647**
Grype wrongly identified CVE-2002-1647 as vulnerable.
The path it identified is:
/usr/share/kibana/node_modules/slash/package.json
The slash npm package version is 3.0.0
However, according to the snyk website, the vulnerability is not related to the slash npm package and according to ubuntu, there are no ubuntu packages affected by the vulnerability with the slash name.

**Comment by willmurphyscode** [comment]
Added `changelog-ignore` because this was fixed in `0.60.0` and so shouldn't be included in the current release's release notes.
